cobbler/items/distro.py

```diff
@@ -155,11 +157,12 @@ def kernel(self, kernel: str):
         """
         if not isinstance(kernel, str):
             raise TypeError("kernel was not of type str")
-        if not utils.find_kernel(kernel):
-            raise ValueError(
-                "kernel not found or it does not match with allowed kernel filename pattern"
-                f"[{utils.re_kernel.pattern}]: {kernel}."
-            )
+        if kernel:
+            if not utils.find_kernel(kernel):
```

Comment:
If I am not mistaken, this should be the same thing, right?

```suggestion
        if kernel and not utils.find_kernel(kernel):
```

Reply:
I think both are equivalent.